doc/configuration.md

```diff
@@ -541,7 +541,7 @@ Supported options:
            * `protocol` - specify protocol for the backend.
            
              Supported protocols:
-               * `carbonapi_v3_pb` - new native protocol, over http. Should be fastest. Currently supported only by [lomik/go-carbon](https://github.com/lomik/go-carbon) and [go-graphite/carbonapi](https://github.com/go-graphite/carbonapi)
+               * `carbonapi_v3_pb` - new native protocol, over http. Should be fastest. Currently supported only by [lomik/go-carbon](https://github.com/lomik/go-carbon), [lomik/graphite-clickhouse](https://github.com/lomik/graphite-clickhouse) and [go-graphite/carbonapi](https://github.com/go-graphite/carbonapi)
```

Comment:
IMO `only` is superfluous here, but I'm nitpicking :)

Reply:
It makes totally sense